src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Errors/ErrorFacts.cs

```diff
@@ -11,132 +11,237 @@ internal static class ErrorFacts
     {
         public static string GetMessage(ErrorCode code)
         {
-            string codeStr = null;
             switch (code)
             {
-                case ErrorCode.ERR_BadBinaryOps: codeStr = SR.BadBinaryOps; break;
-                case ErrorCode.ERR_IntDivByZero: codeStr = SR.IntDivByZero; break;
```

Comment:
I think this is for dumb JITs. If every return emits an individual epilog + ret, numerous returns could be a problem. 
We do not know if we are using a smart enough JIT here. Also sometimes they do not do some of the optimizations when compiling dynamic methods.

I think we should have a single return here.